### ISSUE TITLE
Revisit break buttons

### ISSUE THREAD

**Issue by zapbot** [zaproxy/zaproxy#739]
```
Had various feedback that people are still having problems with the break buttons, so
we really do need to revisit these asap.
I'll start a thread on the dev list...
```

Original issue reported on code.google.com by `psiinon` on 2013-07-18 12:21:02

**Comment by kingthorin** [comment]
Can this issue be closed as duplicate of Issue zaproxy/zaproxy#1353?

**Comment by thc202** [comment]
Closing, addressed by #1353.
